app/dashboard/static/js/app/view-tests-suite-id.2017.7.2.js

```diff
@@ -0,0 +1,631 @@
+        docFrag = document.createDocumentFragment();
+        spanNode = docFrag.appendChild(document.createElement('span'));
+        tooltipNode = spanNode.appendChild(html.tooltip());
+        tooltipNode.setAttribute('title', 'Test reports');
+
+        aNode = tooltipNode.appendChild(document.createElement('a'));
+        str = '/test/';
+        str += board;
+        str += '/job/';
+        str += job;
+        str += '/branch/';
+        str += branch;
+        str += '/kernel/';
+        str += kernel;
+        str += '/defconfig/';
+        str += defconfigFull;
+        str += '/';
+        aNode.setAttribute('href', str);
+        aNode.appendChild(document.createTextNode(defconfigFull));
+
+        if (result.build_id) {
+            spanNode.insertAdjacentHTML('beforeend', '&nbsp;&mdash;&nbsp;');
+
+            tooltipNode = spanNode.appendChild(html.tooltip());
+            tooltipNode.setAttribute('title', 'Build details');
+            aNode = tooltipNode.appendChild(document.createElement('a'));
+            str ='/build/id/';
+            str += result.build_id.$oid;
+            str += '/';
+            aNode.setAttribute('href', str);
+            aNode.insertAdjacentHTML('beforeend', '&nbsp;');
+            aNode.appendChild(html.build());
+        }
+
+        html.replaceContent(
+            document.getElementById('dd-suite-defconfig'), docFrag);
+
+        // Date.
+        docFrag = document.createDocumentFragment();
+        spanNode = docFrag.appendChild(document.createElement('time'));
+        spanNode.setAttribute('datetime', createdOn.toISOString());
+        spanNode.appendChild(
+            document.createTextNode(createdOn.toCustomISODateTime()));
+        html.replaceContent(document.getElementById('dd-date'), docFrag);
+
+        // Status.
+        // TODO Fix when defined
+        html.replaceContent(
+            document.getElementById('dd-suite-status'), html.nonavail());
+
+        // Errors
+        // TODO Fix when defined
+        html.replaceContent(
+            document.getElementById('dd-suite-errors'), html.nonavail());
+
+        // Warnings
+        // TODO Fix when defined
+        html.replaceContent(
+            document.getElementById('dd-suite-warnings'), html.nonavail());
+
+        // Arch.
+        html.replaceContent(
+            document.getElementById('dd-suite-arch'),
+            document.createTextNode(arch));
+
+        // Time.
+        html.replaceContent(
+            document.getElementById('dd-suite-test-time'),
+            document.createTextNode(testTime.toCustomTime()));
+    }
+
+    function getBootLog(response) {
+        var bootLog;
+        var deferred;
+        var pathURI;
+        var result;
+        var serverURI;
+        var serverURL;
+        var translatedURI;
+        var lab;
+        // We only have one result.
+        result = response.result[0];
+        bootLog = null;
+
+        function bootLogDone(bootLog, bootResponse) {
```

Comment:
You are passing `bootLog` that is defined in the outer scope: you can avoid passing it and just access it from inside the function scope.